src/docs/ContrastBox.jsx

```diff
@@ -3,15 +3,15 @@ import PropTypes from 'prop-types';
 import classnames from 'classnames';
 
 const contrastBlockCssClass = 'docs-block__contrast-box';
-const ContrastBox = ({toBottom, smallPadding, light, fullWidth, children}) => {
+const ContrastBox = ({toBottom, smallPadding, light, fullWidth, children, style}) => {
```

Comment:
i think it's not garbage , what is your proposition? adding `width` prop here has no point imho